### PR TITLE
Fix b/25422988 Update Xerces-J to v2.11.0

### PR DESCRIPTION
**Pull request by BrettMichaelJohnson**

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -84,8 +84,8 @@
       <!--include name="resolver-2.9.1.jar"/-->
       <!--include name="serializer-2.9.1.jar"/-->
       <!--include name="xalan-2.7.1.jar"/-->
-      <include name="xercesImpl-2.9.1.jar"/>
-      <!--include name="xml-apis-2.9.1.jar"/-->
+      <include name="xercesImpl-2.11.0.jar"/>
+      <include name="xml-apis-1.4.1.jar"/>
```
Comment (wiarlawd) [inline build.xml:88]: I don't see any record of why these libraries were commented out. I assume you needed this one with 2.11.0 and we didn't with 2.9.1 for some reason? I don't remember adding this when I tested 2.11.0, but I may have just forgotten.

**Review by wiarlawd — APPROVED** [review]

File: build.xml (hunk 1 above)
Comment (BrettMichaelJohnson) [reply build.xml:88]: [junit] Testcase: testBasicHttpsListen took 0.392 sec
    [junit] 	Caused an ERROR
    [junit] org/w3c/dom/ElementTraversal
    [junit] java.lang.NoClassDefFoundError: org/w3c/dom/ElementTraversal
    [junit] 	at java.lang.ClassLoader.defineClass1(Native Method)
    [junit] 	at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
    [junit] 	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
    [junit] 	at java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
    [junit] 	at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
    [junit] 	at java.net.URLClassLoader$1.run(URLClassLoader.java:368)
    [junit] 	at java.net.URLClassLoader$1.run(URLClassLoader.java:362)
    [junit] 	at java.security.AccessController.doPrivileged(Native Method)
    [junit] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:361)
    [junit] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
    [junit] 	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
    [junit] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    [junit] 	at org.apache.xerces.parsers.AbstractDOMParser.startDocument(Unknown Source)
    [junit] 	at org.apache.xerces.impl.dtd.XMLDTDValidator.startDocument(Unknown Source)
    [junit] 	at org.apache.xerces.impl.XMLDocumentScannerImpl.startEntity(Unknown Source)
    [junit] 	at org.apache.xerces.impl.XMLVersionDetector.startDocumentParsing(Unknown Source)
    [junit] 	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
    [junit] 	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
    [junit] 	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source)
    [junit] 	at org.apache.xerces.parsers.DOMParser.parse(Unknown Source)
    [junit] 	at org.apache.xerces.jaxp.DocumentBuilderImpl.parse(Unknown Source)
    [junit] 	at javax.xml.parsers.DocumentBuilder.parse(DocumentBuilder.java:121)
    [junit] 	at org.apache.xml.security.Init.init(Unknown Source)
    [junit] 	at org.opensaml.DefaultBootstrap.initializeXMLSecurity(DefaultBootstrap.java:110)
    [junit] 	at org.opensaml.DefaultBootstrap.bootstrap(DefaultBootstrap.java:84)
    [junit] 	at com.google.enterprise.adaptor.GsaCommunicationHandler.bootstrapOpenSaml(GsaCommunicationHandler.java:657)
    [junit] 	at com.google.enterprise.adaptor.GsaCommunicationHandler.start(GsaCommunicationHandler.java:277)
    [junit] 	at com.google.enterprise.adaptor.Application.realDaemonStart(Application.java:200)
    [junit] 	at com.google.enterprise.adaptor.Application.daemonStart(Application.java:150)
    [junit] 	at com.google.enterprise.adaptor.Application.start(Application.java:105)
    [junit] 	at com.google.enterprise.adaptor.ApplicationTest.testBasicHttpsListen(ApplicationTest.java:248)

**Comment by BrettMichaelJohnson** [comment]
PJ, do you have any input to this change?

**Review by PapaJoltOmega — APPROVED** [review]
Thank you

**Pull request merged**